integration/test/API/Cargohold.hs

```diff
@@ -15,24 +14,6 @@ import UnliftIO (catch)
 
 type LByteString = LBS.ByteString
 
-getFederationAsset :: (HasCallStack, MakesValue asset) => asset -> App Response
```

Comment:
have you checked that our asset uploads have reasonable coverage? this could be an indication that they're not, or there could be a better way to test the API.